lib/commands/dump.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+module.exports = {
+	/**
+	 * @param {Object} data
+	 * @param {Object} data.parameters Additional arguments provided by the user.
+	 * @param {String} data.packageName Name of current package to process.
+	 * @param {Options} data.options The options object.
+	 * @param {Repository|null} data.repository
+	 * @returns {Promise}
+	 */
+	execute( data ) {
+		const log = require( '../utils/log' )();
+		const execCommand = require( './exec' );
+
+		return new Promise( ( resolve, reject ) => {
+			execCommand.execute( getExecData( 'git rev-parse HEAD' ) )
+				.then( ( execResponse ) => {
+					const commitHash = execResponse.logs.info[ 0 ];
+
+					const commandResponse = {
+						packageName: data.packageName,
+						commit: commitHash
+					};
+
+					log.info( `Commit: ${ commitHash }.` );
+
+					resolve( {
+						response: commandResponse,
+						logs: log.all()
+					} );
+				} )
+				.catch( ( error ) => {
+					log.error( error );
+
+					reject( { logs: log.all() } );
+				} );
+		} );
+
+		function getExecData( command ) {
+			return Object.assign( {}, data, {
+				parameters: [ command ]
+			} );
+		}
+	},
+
+	/**
+	 * Saves collected hashes to configuration file.
+	 *
+	 * @param {Set} processedPackages Collection of processed packages.
+	 * @param {Set} commandResponses Results of executed command for each package.
+	 */
+	afterExecute( processedPackages, commandResponses ) {
+		const cwd = require( path.join( __dirname, '..', 'utils', 'getcwd.js' ) )();
```

Comment:
I guess so. Otherwise `require` will search the `utils/getcwd.js` file based on current work directory.

```js
console.log('Path for utils (based on cwd):', path.resolve( 'utils', 'getcwd.js' ));
//Path for utils (based on cwd): /Users/pomek/Projects/reinmar/ckeditor5/utils/getcwd.js

console.log('Path for utils (based on current file path):', path.join( __dirname, '..', 'utils', 'getcwd.js' ));
//Path for utils (based on current file path): /Users/pomek/Projects/cksource/mgit2/lib/utils/getcwd.js
```